prod/commonvars.hcl

```diff
@@ -14,4 +14,9 @@ locals {
 
   service_api_url = "http://api-internal.io.italia.it/"
 
+  # Switch limit date for email opt out mode. This value should be used by functions that need to discriminate
+  # how to check isInboxEnabled property on IO profiles, since we have to disable email notifications for default
+  # for all profiles that have been updated before this date. This date should coincide with new IO App's release date
+  opt_out_email_switch_date = "2021-07-09T00:00:00Z"
```

Comment:
I remember a bug into app service api using date in ISO format.
Better use timestamp

Reply:
Refactor done in [8a46c34](https://github.com/pagopa/io-infrastructure-live-new/pull/575/commits/8a46ce4644a63b1a43117cc40494c033400cbad2). We must wait for https://github.com/pagopa/io-functions-services/pull/140 and https://github.com/pagopa/io-functions-app/pull/180 before applying :)